experiment_manager.py

```diff
@@ -351,18 +365,26 @@ def sweep(self, config):
         outer_variables = Config(*variable_config.param_list[:-1])  # "outer" means all but the inner-most loop
         # N-dimensional loop with itertools.product: # (actually N-1 )
 
+        counter = 1
+        num_jobs = 1
+        for var in outer_variables.param_list:
+            num_jobs=num_jobs*len(var.value)
```

Comment:
This look like a method of the `Config` class

Reply:
fixed